src/microbe_stage/Microbe.Interior.cs

```diff
@@ -632,6 +638,19 @@ public float EjectCompound(Compound compound, float maxAmount, Vector3 direction
         return result;
     }
 
+    /// <summary>
+    ///   Returns the check result whether this microbe can digest the target (has the enzyme necessary).
+    /// </summary>
+    public DigestCheckResult CanDigestObject(IEngulfable engulfable)
```

Comment:
Isn't there already a really similar method existing for checking if something can be engulfed?

Reply:
Yes but it doesn't check whether something can be _digested_.

Reply:
I see, could you update the documentation on this method and that other one to refer to each other and explain the difference?

Reply:
Done.